src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoTests.cs

```diff
@@ -131,7 +131,35 @@ public void ShortestDayNamesTest(string cultureName)
             for (DayOfWeek day=DayOfWeek.Sunday; day <= DayOfWeek.Saturday; day++)
             {
                 Assert.Equal(shortestDayNames[(int) day], dtfi.GetShortestDayName(day));
-            } 
+            }
+        }
+
+        [Fact]
+        public void TestHebrewMonths()
```

Comment:
nit: optionally I'd rename it to something with RoundTrip in the name

Reply:
I am not testing round tripping more than ensuring we can parse back the formatted string.

Reply:
Isn't that what round-tripping means? (I always thought of it as testing `X=F(F^(-1)(X))`)

Reply:
The test mainly interested in the Hebrew months handling, getting it from DTFI, formatting it and parse it. it doesn't matter if we are doing round tripping in here. I want to have the test name reflect what is the intention of what we are trying to test.